### PR TITLE
Add nsyilxcən to g2p, language code oka

### PR DESCRIPTION
**Pull request by joanise**
Pull request template for adding a new language
-----------------------------------------------

*Do not use this for other types of pull requests*

* **Please check if the PR fulfills these requirements**
- [X] Mapping files are added in `g2p/mappings/langs`
- [X] Mapping is either added to an existing folder or a new folder has been added
- [X] Language folder and files use appropriate [ISO 639-3 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-3_codes)
- [X] Config.yaml file includes all author names, and settings necessary
- [X] Please add some test data in `g2p/tests/public/data`. The added file should be a csv/tsv/psv file and each row should have the format `[input_mapping_code,output_mapping_code,input_string,output_string]`
- [X] As the last step, G2P has been updated by running `g2p update` locally and committing the change
- [X] You agree to license your contribution under the same license as this project (see [LICENSE](https://github.com/roedoejet/g2p/blob/master/LICENSE) file).

* **Other information**:

### CODE REVIEW

**Comment by joanise** [comment]
@roedoejet This has now been approved by the community and is ready to merge.

I've validated it and fixed what needed fixing.

File: g2p/mappings/langs/oka/oka_equiv.csv [hunk 1]
```diff
@@ -0,0 +1,15 @@
+’,'
+‘,'
```
Comment (roedoejet) [inline g2p/mappings/langs/oka/oka_equiv.csv:2]: would be nice to have a standard list of confusable apostrophes here that we could just extend from

**Review by roedoejet — APPROVED** [review]
Fantastic! Well done Craig & Eric!!

File: g2p/mappings/langs/oka/oka_equiv.csv (hunk 1 above)
Comment (joanise) [reply g2p/mappings/langs/oka/oka_equiv.csv:2]: That's true, we keep having to recreate this list!

**Pull request merged**